packages/language-server/test/snapshots/xml-view-diagnostics/use-of-deprecated-attribute/input.xml

```diff
@@ -0,0 +1,9 @@
+<mvc:View xmlns:m="sap.m" 
+  xmlns:mvc="sap.ui.core.mvc">
+  <m:Page 🢂navButtonText🢀="dummy-text">
```

Comment:
can we locate and add here a scenario of an aggregation with `alttypes`a.k.a aggregation as attribute?

Reply:
See: https://openui5nightly.hana.ondemand.com/#/api/sap.ui.table.Table/aggregations

![image](https://user-images.githubusercontent.com/4233843/84672309-1479b400-af31-11ea-9a69-de976b77c3f9.png)

Reply:
maybe it would be a problem to find such a scenario because it needs to be both deprecated and have an alt-type

Reply:
All aggregations can be used as attributes for binding. You don't need to find one with `altType`.

Reply:
ah cool